src/spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -187,9 +185,17 @@ def __init__(
                     self.set_probe(probe, in_place=True, group_mode="by_shank")
                 else:
                     self.set_probe(probe, in_place=True)
-                probe_name = probe.annotations["probe_name"]
+
+                # this try-except handles a breaking change in probeinterface after v0.2.18
+                # in the new version, the Neuropixels model name is stored in the "model_name" attribute,
+                # rather than in the "probe_name" annotation
+                try:
+                    model_name = probe.model_name
+                except Exception as e:
+                    model_name = probe.annotations["probe_name"]
```

Comment:
Lets do this without try except please.

Reply:
We can't use the version since we don't have dev in probeinterface...